smoke/IotEdgeQuickstart/Quickstart.cs

```diff
@@ -49,9 +52,12 @@ public async Task RunAsync()
             // its config. This could happen, for example, if someone were to create an at-scale deployment on the
             // test hub with the target condition: "NOT deviceId=''". Since this is an unlikely scenario, we won't
             // invest the effort to guard against it.
-            await this.VerifyEdgeIsNotAlreadyActive(); // don't accidentally overwrite an edge configuration on a dev machine
-            await this.VerifyBootstrapperDependencies();
-            await this.InstallBootstrapper();
+            if (!this.bypassEdgeInstallation)
+            {
+                await this.VerifyEdgeIsNotAlreadyActive(); // don't accidentally overwrite an edge configuration on a dev machine
+                await this.VerifyBootstrapperDependencies();
+                await this.InstallBootstrapper();
```

Comment:
This comment "Windows installation does install + configure in one step. Since we need to connection string to configure and we don't have that information here, we'll do installation in Configure()." is extracted from IotedgedWindows.cs.  If you want to skip edge installation, you need to refactor the code to split install and config logic.

Reply:
Separation is done in _IotedgedWindows.cs_, function _Configure_, via the _readonly bool isCleanInstall_ member of the _IotedgedWindows_ class.
Is that what you mean?